### ISSUE TITLE
Being told to upgrade to a version I'm on

### ISSUE THREAD

**Issue by williamhammond** [stripe/stripe-cli#241]
_The more information we have the easier it is for us to help. Feel free to remove any sections that might not apply_

### Issue
_Describe what happened and what you were trying to do_

I'm attempting to generate a `sigma.scheduled_query_run.created` event. After installing the CLI I attempt to run the command and see. 
```
 williamhammond$ stripe trigger sigma.scheduled_query_run.created
A newer version of the Stripe CLI is available, please update to: v0.7.3
event sigma.scheduled_query_run.created is not supported.
williamhammond$ stripe --version
stripe version 0.7.3 (beta)
```
After looking at the available versions I appear to have the latest and this is telling me to upgrade to the version I'm on. 

### Expected Behavior
_Tell us what you expected to happen_

I'd expect the event to be created or atleast not show the message about upgrading to the version I'm on. 

### Steps to reproduce
_What are the steps we can take to reproduce this and verify it's fixed?_

1. `brew install stripe/stripe-cli/stripe`
2. `stripe login`
3. `stripe trigger sigma.scheduled_query_run.created`

### Environment
_Select one of: macOS, Linux, Windows, Other_
macOS Mojave 10.14.6

**Comment by williamhammond** [comment]
Sorry for kind of conflating two issues if you'd like I can make two tickets. I tested on master and I'm still seeing 
```
C02V30R4HV2H:stripe-cli williamhammond$ stripe-dev trigger sigma.scheduled_query_run.created
event sigma.scheduled_query_run.created is not supported.
exit status 1
```
even though https://github.com/stripe/stripe-cli/blob/4fa4bd94e8cabaada17a8be5af3cc1ca4bf6b7a8/pkg/cmd/events_list.go#L125 implies that it is a supported event type

**Comment by ob-stripe** [comment]
Hi @williamhammond. We've just released v0.7.4 which should hopefully solve the invalid version upgrade message. Can you try upgrading and see if you're still observing the issue?

For the other issue (unable to trigger `sigma.scheduled_query_run.created`), it'd be preferable to open a separate ticket, but I can confirm that right now the CLI cannot trigger this event type.

**Comment by williamhammond** [comment]
Cool thank you!
